altdoc/quarto_website.yml

```diff
@@ -36,12 +35,15 @@ website:
             file: man/draw_legend.qmd
         - section: Plot types
           contents:
-          #
           - section: Shapes
             contents:
-            - text: type_area, type_ribbon
```

Comment:
I think this change should be revert since area plots are a special case of ribbon plots. The underlying docs point to the same help file.

Reply:
I can revert if you like. I originally changed it because it hurts readability in the side bar. And since we're going to have *a lot* of types...

<img width="264" alt="Screenshot 2024-11-13 at 6 53 22 AM" src="https://github.com/user-attachments/assets/f2cab6f6-907c-4c3c-be91-07b0093c11f4">